Tasks/NuGetCommandV2/nugetpublisher.ts

```diff
@@ -466,4 +483,37 @@ function getAccessToken(isInternalFeed: boolean): string{
     }
 
     return accessToken;
-}
\ No newline at end of file
+}
+
+async function tryServiceConnection(endpoint: EndpointCredentials, feed: any) : Promise<boolean>
+{
+    // Create request
+    const request = new WebRequest();
+    const token64 = Buffer.from(`${endpoint.username}:${endpoint.password}`).toString('base64');
+    request.uri = endpoint.endpoint;
+    request.method = 'GET';
+    request.headers = {
+        "Content-Type": "application/json",
+        "Authorization": "Basic " + token64
+    };
+
+    return await sendRequest(request).then((response: WebResponse) => {
+        if(response.statusCode == 200) { // Not sure if we actually need this null check here..
+            if(response.body) {
+                for (const entry of response.body.resources) {
+                    if (entry['@type'] === 'AzureDevOpsProjectId' && entry['label'].toUpperCase() !== feed.projectId.toUpperCase()) 
+                    {
+                        return false;
+                    }
+                    if (entry['@type'] === 'VssFeedId' &&  entry['label'].toUpperCase() !== feed.feedId.toUpperCase())
```

Comment:
Is there a better way of doing this case-sensitive check? I'm unsure if this will capture all possible combinations a user may have for their feed name

Reply:
[localeCompare](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare) appears to be the correct locale-sensitive comparison function, at least for matching whole strings against each other

Reply:
That said, toUpperCase is probably good enough. Seems to be ~99.9% of the relevant advice on the subject for JS unfortunately.